content/reference/admin/private_locations/configuration/kubernetes/index.md

```diff
@@ -115,7 +115,7 @@ control-plane {
       #   "--add-opens=java.base/jdk.internal.misc=ALL-UNNAMED"
       # ]
       # Based on your instance configuration, you may want to update Xmx and Xms values.
-      jvm-options = ["-Xmx8G", "-Xms512M"]
+      # jvm-options = ["-Xmx4G", "-Xms512M"]
```

Comment:
Do we provide default values? If not, IIRC, Xmx is the smallest of 1/4th of the physical memory or 1GB. Not sure what `physical memory` is on k8s: mem request or mem limit? I'm concern we'd end up with 256MB or less. Shouldn't we set 1GB like on Gatling OSS?

Reply:
Default configuration available at https://github.com/gatling/frontline-cloud/blob/dev/back/hexagon/run/port/model/src/main/scala/io/gatling/enterprise/hexagon/run/model/HardcodedConfiguration.scala#L14